packages/playwright-core/src/server/trace/recorder/snapshotterInjected.ts

```diff
@@ -59,6 +60,12 @@ export function frameSnapshotStreamer(snapshotStreamer: string) {
     cssRef?: number, // Previous snapshotNumber for overridden stylesheets.
   };
 
+  function maskIfSecret(value: string) {
+    if ((window as any).__playwright_secrets_?.has(value))
+      return maskString(value);
```

Comment:
I don't think we can use imported functions in `snapshotterInjected`, because we do not build+bundle it.